src/Aquifer.Data/Migrations/AquiferDbContextModelSnapshot.cs

```diff
@@ -2718,7 +2718,7 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.HasOne("Aquifer.Data.Entities.VerseEntity", "BibleVerse")
                         .WithMany()
                         .HasForeignKey("BibleVerseId")
-                        .OnDelete(DeleteBehavior.Cascade)
+                        .OnDelete(DeleteBehavior.NoAction)
```

Comment:
should I do this for other Verse keys just in case?

Reply:
This is a generated file, shouldn't be updating it directly. Need to do it like in `ResourceContentVersionAssignedUserHistoryEntity.cs`, and redo the migrations.